multimodal/src/autogluon/multimodal/utils/config.py

```diff
@@ -761,3 +774,20 @@ def filter_hyperparameters(
         hyperparameters = {k: v for k, v in hyperparameters.items() if not k.startswith(key)}
 
     return hyperparameters
+
+
+def split_hyperparameters(hyperparameters: Dict):
+    if not hyperparameters:
+        return dict(), dict()
+
+    advanced_hyperparameters = dict()
+    for k, v in hyperparameters.items():
+        if k in [
+            "model.timm_image.train_transforms",
+            "model.timm_image.val_transforms",
+        ]:
+            if not all([isinstance(trans, str) for trans in hyperparameters[k]]):
+                advanced_hyperparameters[k] = copy.deepcopy(v)
+                hyperparameters[k] = []
```

Comment:
After split, the hyperparameter dict itself doesn't aware whether it has been splitted or not. Basically, some advanced hyperparameters are modified (not deleted), and copied to another variable, where the naive hyperparameters don't have access to.

Since hyperparameter space has been divided into subspaces with the dot delimitor, how about moving advanced hyperparameters into 'advanced.xxx' subspace ?? Potentially, with OmegaConf, we can easily access to these advanced hyperparameters with `config.advanced`.